requirements-test.txt

```diff
@@ -13,3 +13,4 @@ openwisp-monitoring @ https://github.com/openwisp/openwisp-monitoring/tarball/ma
 django-redis~=5.4.0
 mock-ssh-server~=0.9.1
 channels_redis~=4.2.1
+chardet
```

Comment:
Is this dependency used only in tests? Can you point out where it is used? Pelase also specify the version as in the lines above.

Reply:
It is used for detecting the file format,
https://github.com/akhilsharmaa/openwisp-radius/blob/01fe5c4e2273cfd915d331825ca0686c636880b9/openwisp_radius/utils.py#L141